client/me/security-ssh-key/manage-ssh-keys.tsx

```diff
@@ -46,8 +64,16 @@ const SSHKey = ( { userLogin, sshKey, onDelete, keyBeingDeleted }: SSHKeyProps )
 				</SSHKeyCard.Date>
 			</SSHKeyCard.Details>
 			<SSHKeyCard.Button
+				primary={ true }
```

Comment:
This button shows up as red for me...

<img width="1093" alt="image" src="https://user-images.githubusercontent.com/36432/201771001-55a7d940-140a-4eab-b3dd-372ef9ff337d.png">

We should probably make 'Remove SSH key' the primary action, and 'Update SSH key' the secondary.